bin/generate-nav

```diff
@@ -1,19 +1,33 @@
 #!/usr/bin/env node
 
+const {docPagesDirectory, outlinePath, current, versions} = require('../constants');
 const fileProcessor = require('../src/lib/fileProcessor');
-const { readFileSync, writeFile } = require('fs');
-const { resolve } = require('path');
+const {readFileSync, writeFile, readdirSync} = require('fs');
+const {getPrefixedVersion} = require('../src/lib/versionHelper');
+const {resolve} = require('path');
 const jsyaml = require('js-yaml');
 
-const loadedOutline = jsyaml.safeLoad(readFileSync(`${__dirname}/../config/outline.yaml`, 'utf8'));
-const docs = { chapters: [] };
+const loadedOutline = jsyaml.safeLoad(readFileSync(outlinePath, 'utf8'));
 
-loadedOutline.chapters.forEach((directory) => {
-    const items = directory.items.map(
-        file => fileProcessor.processFile(directory.path, file.id)[0]
-    );
+versions.push(current);
+versions.forEach(version => {
+    let docs = { chapters: [] };
+    const versionDirectory = getPrefixedVersion(version);
 
-    docs.chapters.push({ ...directory, items });
-});
+    docs.chapters = loadedOutline.chapters.map(directory => {
+        let files = readdirSync(`${docPagesDirectory}${versionDirectory}/${directory.path}`)
```

Comment:
recommendation here would be to use `path.join` or even `path.resolve` (https://nodejs.org/api/path.html#path_path_join_paths)